tests/app/dao/test_fact_notification_status_dao.py

```diff
@@ -703,3 +703,39 @@ def test_fetch_monthly_notification_statuses_per_service_for_rows_that_should_be
 
     results = fetch_monthly_notification_statuses_per_service(date(2019, 3, 1), date(2019, 3, 31))
     assert len(results) == 0
+
+
+def test_get_total_notifications_for_date_range(sample_service):
+    template_sms = create_template(service=sample_service, template_type='sms', template_name='a')
+    template_email = create_template(service=sample_service, template_type='email', template_name='b')
+    template_letter = create_template(service=sample_service, template_type='letter', template_name='c')
+    create_ft_notification_status(bst_date=date(2021, 2, 28),
+                                  service=template_email.service,
+                                  template=template_email,
+                                  count=15)
+    create_ft_notification_status(bst_date=date(2021, 2, 28),
+                                  service=template_sms.service,
+                                  template=template_sms,
+                                  count=20)
+    create_ft_notification_status(bst_date=date(2021, 2, 28),
+                                  service=template_letter.service,
+                                  template=template_letter,
+                                  count=3)
+
+    create_ft_notification_status(bst_date=date(2021, 3, 1),
+                                  service=template_email.service,
+                                  template=template_email,
+                                  count=15)
+    create_ft_notification_status(bst_date=date(2021, 3, 1),
+                                  service=template_sms.service,
+                                  template=template_sms,
+                                  count=20)
+    create_ft_notification_status(bst_date=date(2021, 3, 1),
```

Comment:
Optional: This is better with the data for the 2021/2/28 but we still aren't testing the upper bound on time. If you added in some data for 2021/3/2 then it would test this. Not the end of the world and I can survive with the gap in test coverage but thought I'd point it out

Reply:
I've added more data to the test